### PR TITLE
feat: Toggle bury

### LINKED ISSUE #14163: Add option to bury/unbury cards in browser
Please add the ability to bury/unbury cards in the card browser.

The desktop version of Anki added a similar feature some time back: https://github.com/ankitects/anki/issues/2295

### PR DESCRIPTION
**Pull request by david-allison**
## Fixes
* Fixes #14163

## Approach
Basically the same as 'suspend', but with a snackbar as feedback

## How Has This Been Tested?
Unit tested

<details>

<summary>Screenshots</summary>

image
image
image
image

</details>

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by lukstbit — APPROVED** [review]
General LGTM
Two notes:
- we already have an icon that was used for bury in reviewer
Screenshot from 2024-05-01 20-29-31
- this doesn't fully replicate the desktop behavior of showing the bury status for the selection in the form of the checked/unchecked box

**Comment by david-allison** [comment]
I believe Desktop only shows the status of the first card, and we're explicitly out-of-alignment here

**Comment by lukstbit** [comment]
Why closing?

**Comment by david-allison** [comment]
Thanks for the catch! Was cleaning up my 200+ old branches in Android Studio and missed that this one was still active

**Comment by lukstbit** [comment]
There's also the one with the removal of the comments from translated strings.

**Comment by david-allison** [comment]
I'm at very limited capacity this week, and don't feel it's high enough priority to get through.

I'll tag it as 'needs new dev' for now, and will get back to it if/when I have time

File: AnkiDroid/src/main/res/values/02-strings.xml [hunk 1]
```diff
@@ -433,5 +433,16 @@ opening the system text to speech settings fails">
 
     <string name="card_browser_unavailable_when_notes_mode">Unavailable in ‘Notes’ mode</string>
 
+    <!-- similar to 'card_buried' -->
+    <plurals name="bury_cards_feedback">
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/02-strings.xml:437]: Why not `CollectionManager.TR.studyingCardsBuried()`

Comment (david-allison) [reply AnkiDroid/src/main/res/values/02-strings.xml]: FIxed

**Review by BrayanDSO — APPROVED** [review]

**Comment by mikehardy** [comment]
strings are in sync again again so hopefully windows CI behaves and this merges

**Pull request merged**